runtime/litentry/src/migration.rs

```diff
@@ -0,0 +1,197 @@
+// Copyright 2020-2021 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
The migration generally looks good! Two small things:
- we should compile it once with `try-runtime` features on to make sure it goes through, I see some potential compilation errors like `<Vec<AccountId>>` in `pre_upgrade`. Eventually we should run try-runtime once with rococo/litmus
- we can add more checks in post_upgrade, like the round count, the number of candidates, total stakes etc